docs/topics_5/proc_web-downloading-logs-cli.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_web-downloading-logs-cli_{context}"]
+= Downloading pod logs using the CLI
+[role="_abstract"]
```

Comment:
What is this? It looks like something from the template, maybe for PV2. I am not familiar with it.